### DISCUSSION TITLE
Cannot respond to follow request.

### DISCUSSION THREAD

**Discussion by nerdteacher** [bookwyrm-social/bookwyrm#2233] [Q&A]
![image](https://user-images.githubusercontent.com/77229429/179404826-ba84bff8-cc6f-41dc-aae3-4fffbdf03d8f.png)

I have this follow request that I cannot respond to, but it always leads to this page (regardless of if I hit "accept" or "deny"). It always goes to this page:

![image](https://user-images.githubusercontent.com/77229429/179404878-af57c5b6-006c-4313-ae84-847b8a52caea.png)

I'm not sure if others have had this issue (I did try searching around) or what might be causing it (e.g., if they decided to cancel their request? not sure). I just can't make it go away, haha.

**Comment by mouse-reeve** [comment]
Yikes, thanks for reporting this! I created an issue from this, since it's a bug with a concrete fix ❤️
